code/data_structures/src/tree/binary_tree/binary_tree/traversal/zigzag/zigzag.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include <stack>
+
+class Node
+{
+    public:
+        int info;
+        Node* left_child;
+        Node* right_child;
+        Node (int info)
+        {
+            this->info = info;
```

Comment:
Should use member initialization list here.

Reply:
Actually, yes, you should use member initializer lists, as that is the general recommended practice.

Reply:
It's not present in C++ style guide, maybe we should add it ?

Reply:
True, you can consider making an addition to generally use initializer lists whenever possible. Good idea!